lib/sippy_cup/scenario.rb

```diff
@@ -149,13 +149,14 @@ def build(steps)
     # @param [Hash] opts A set of options to modify the message
     # @option opts [Integer] :retrans
     # @option opts [String] :headers Extra headers to place into the INVITE
+    # @option opts [String] :remote_sip_address Remote address to use in place of the SIPp constructed one
```

Comment:
There are things that SippyCup should probably never try to do. The goal of SippyCup is to make SIPp manageable. If we go on to support every permutation of possibilites that SIPp allows, we're going to end up in a deep hole.

@benlangfeld can you tell me how often you anticipate running into this issue, where the remote SIP address differs from the address to which packets are sent? And if so, is this not already handled by having the `invite` instruction take an address which may be different from the address specified in the general test options?